frontend/src/components/general-comparison-table/general-comparison-table.tsx

```diff
@@ -80,31 +117,46 @@ const GeneralComparisonTable: React.FC = () => {
       value.style.height = `${optionsHighestHeights.get(title)}px`;
       tableTitleElement.style.height = `${optionsHighestHeights.get(title)}px`;
     });
-  }, [generalTable, generalInfo]);
+  }, [generalTable, generalInfo, toggle]);
 
   if (isLoading) return <Spinner />;
 
   return (
     <CollapseElement label="General information" isOpen={true}>
       <div className={styles.table} ref={setGeneralTableRef}>
         <div className={clsx(styles.tableTitles, styles.tableColumn)}>
-          <div className={styles.tableCell} data-optiontitle="bodytype">
-            Type
-          </div>
-          <div className={styles.tableCell} data-optiontitle="motor">
-            Motor
-          </div>
-          <div className={styles.tableCell} data-optiontitle="enginepower">
-            Engine Power
-          </div>
-          <div className={styles.tableCell} data-optiontitle="engine">
-            Engine
-          </div>
-          <div className={styles.tableCell} data-optiontitle="wheeldrive">
-            Wheel Drive
-          </div>
+          {(toggle && compareFields.bodyType) || (
+            <div className={styles.tableCell} data-optiontitle="bodytype">
+              Type
+            </div>
+          )}
```

Comment:
> If toggle is `true` -> show `compareFields.bodyType`, and if not -> show `div`

Do I understand correctly what it does?

Because if it's correct, I believe a simple `?:` operator would be much easier to comprehend

Reply:
Nope, I use brackets `()` and `&&` here so when both `toggle` and `compareFields.bodyType` are `true` -> hide `div`. In all other situations show `div`. 

`toggle` = `true` means we're in `only difference` mode now. 
`compareFields.bodyType` = `true` means bodyType value for all cars is the same. (Maybe I need to name it more explicit)

Reply:
> toggle = true means we're in only difference mode now.

Then the `toggle` is extremely misleading name

Reply:
> hide div. In all other situations show div.

`||` is an extremely unintuitive way to do that! Please rewrite it with the `&&`: `{(!isOnlyDiff || !compareFields.bodyType) && <div>...</div>}`